pkg/parser/parser.go

```diff
@@ -49,6 +52,10 @@ func List() []string {
 		supportedParsers = append(supportedParsers, p)
 	}
 
+	for p, _ := range customExperimentalParsers {
+		supportedParsers = append(supportedParsers, p)
```

Comment:
If we are going to generally support these parsers then we need to update the test case here:
https://github.com/safedep/vet/blob/main/pkg/parser/parser_test.go#L11

Originally I did not add the pywheel or any experimental parser in this list so that they are not automatically used while directory scanning. They however can be used explicitly by:

```
--lockfiles python-wheel
--lockfiles cydx-sbom
```

I feel we should explicitly use these experimental parsers and not add them into the list of `supportedParsers`

Reply:
How will users get to know that these parsers are available? there is no way to get the names unless an user look into the code